python/ray/serve/experimental/gradio_visualize_graph.py

```diff
@@ -232,6 +234,22 @@ def depths_fn(node):
         }
 
         with gr.Blocks() as demo:
+            from PIL import Image
+
+            try:
+                graph = _dag_to_dot(self.dag)
+                gr.Image(
+                    label="Ray Serve Deployment Graph",
+                    value=Image.open(BytesIO(graph.create(graph.prog, format="png"))),
+                )
+            except ImportError:
+                logger.warning(
+                    "Python module `pydot` and package `graphviz` is needed to show "
+                    "illustration of graph. Install pydot with `pip install pydot` "
+                    "and graphviz with either `brew install pydot` or `sudo apt "
+                    "install graphviz`"
+                )
```

Comment:
can this be a gr.Label to surface to ewb ui directly?

Reply:
Changed to a gr.Markdown that shows the warning